### PR TITLE
Adding a new recipe

### PR DESCRIPTION
**Pull request by ojulio-dev**
Create a new recipe (Creme de Açaí).

### CODE REVIEW

File: src/products.json [hunk 1]
```diff
@@ -167,5 +167,48 @@
...
+    "images": [
+      {
+        "position": 1,
+        "src": "https://creamberry.com.br/wp-content/uploads/2021/02/acai-2709077_1920-1024x683.jpg"
+      }
+    ],
+    "ingredients": [
+      {
+        "position": 1,
+        "name": "300 ml de açaí ou suco de açaí"
```
Comment (emanuelcorrea) [inline src/products.json]: Please, add a dot allways in the end of each ingredient name.

File: src/products.json [hunk 2]
```diff
@@ -167,5 +167,48 @@
...
+      },
+      {
+        "position": 2,
+        "name": "50 ml de xarope de guaraná"
```
Comment (emanuelcorrea) [inline src/products.json]: Please, add a dot allways in the end of each ingredient name.

File: src/products.json [hunk 3]
```diff
@@ -167,5 +167,48 @@
...
+      },
+      {
+        "position": 3,
+        "name": "50 ml de leite"
+      },
+      {
+        "position": 4,
+        "name": "1 colher de açúcar"
```
Comment (emanuelcorrea) [inline src/products.json]: Please, add a dot allways in the end of each ingredient name.

File: src/products.json [hunk 4]
```diff
@@ -167,5 +167,48 @@
...
+        "position": 1,
+        "name": "300 ml de açaí ou suco de açaí"
+      },
+      {
+        "position": 2,
+        "name": "50 ml de xarope de guaraná"
+      },
+      {
+        "position": 3,
+        "name": "50 ml de leite"
```
Comment (emanuelcorrea) [inline src/products.json]: Please, add a dot allways in the end of each ingredient name.

File: src/products.json [hunk 5]
```diff
@@ -167,5 +167,48 @@
...
+      },
+      {
+        "position": 4,
+        "name": "1 colher de açúcar"
+      },
+      {
+        "position": 5,
+        "name": "3 conchas de gelo moído"
```
Comment (emanuelcorrea) [inline src/products.json]: Please, add a dot allways in the end of each ingredient name.

File: src/products.json [hunk 6]
```diff
@@ -167,5 +167,48 @@
...
+      }
+    ],
+    "preparations": [
+      {
+        "position": 1,
+        "name": "Bata no liquidificador o açaí ou suco de açaí, o xarope de guaraná, o leite e o açúcar."
```
Comment (emanuelcorrea) [inline src/products.json:206]: Please, add a dot allways in the end of each ingredient name.

File: src/products.json [hunk 7]
```diff
@@ -167,5 +167,48 @@
...
+      },
+      {
+        "position": 2,
+        "name": "Coloque o gelo aos poucos, até ele ficar bem consistente."
```
Comment (emanuelcorrea) [inline src/products.json:210]: Please, add a dot allways in the end of each ingredient name.

File: src/products.json [hunk 8]
```diff
@@ -167,5 +167,48 @@
...
+      }
+    ]
   }
 ]
```
Comment (emanuelcorrea) [inline src/products.json]: Could you add a empty line in the end of file?

**Review by emanuelcorrea — CHANGES_REQUESTED** [review]
Review

**Review by ojulio-dev — COMMENTED** [review]
Fix request changes

**Review by emanuelcorrea — APPROVED** [review]
Request approved.

**Pull request merged**